src/js/entities-service/entities/clinicians.js

```diff
@@ -24,7 +24,11 @@ const _Model = BaseModel.extend({
     return Radio.request('entities', 'roles:model', this.get('_role'));
   },
   can(prop) {
-    return this.get('access') === 'manager';
+    /* istanbul ignore next */
+    return (_DEVELOP_ && !sessionStorage.getItem('cypress'))
+      || this.get('access') === 'manager'
```

Comment:
also this is super simplistic and not very good.  Left it in this state as a clear "we're definitely going to refactor this at some point once we have more use cases" kind of thing.